templates/appPackage/manifest.local.template.json

```diff
@@ -40,6 +40,10 @@
             {
               "title": "learn",
               "description": "Learn about Adaptive Card and Bot Command"
+            },
+            {
+              "title": "start office hours",
```

Comment:
I started running the app locally and was not able to get initialize office hours using the command. I think what may be happening here is that the switch case in teamsBot.ts is looking for input of "start office hour", while this remote template gives a menu option as "start office hour**s**". Manually typing in the command "start office hour" does initialize the queue and the bot responds. Fixing this should allow for the command in the menu to work!

Reply:
Thanks for the heads up! I fixed this in commit e432cc4 but it didn't do the trick for the new command. I found a way to force the update, which I've written out in https://github.com/beaver-hours/BeaverHours/pull/12#issuecomment-1015000853.